snippets/toCurrency.md

```diff
@@ -0,0 +1,17 @@
+### toCurrency
+
+Take a number and return specified currency formatting.
+
+Use `new Intl.NumberFormat` to enable country / currency sensitive formatting.
+
+```js
+const toCurrency = (n, curr, LanguageFormat = undefined) => new Intl.NumberFormat(LanguageFormat, { style: 'currency', currency: curr }).format(n);
```

Comment:
thew `new` keyword is unneeded. NumberFormat is a static class function

Reply:
Although omitting/removing `new` isn't much. I guess it does help make snippet shorter.  Change's made @skatcat31 😄

Reply:
depends on whether it will properly optimize in engine, however it still calls a bit slower, so in loops where you'd use this is can slow it down significantly